recipes/hypothesis/meta.yaml

```diff
@@ -0,0 +1,47 @@
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set version = "3.4.2" %}
+{% set md5 = "b180f47bdbebdbbbfe4a6d2d7f95c640" %}
+{% set pkgname = "hypothesis " %}
+
+package:
+  name: {{ pkgname }}
+  version: {{ version }}
+
+source:
+  fn: simplejson-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ pkgname[0] }}/{{ pkgname }}/{{ pkgname }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - django
+    - enum34 [py2k or py < 34]
```

Comment:
So we prefer the selector to use the comment syntax and have two spaces as the linter has noted. Though I would also simplify this selector as `py<34` includes `py2k`. So would change this to the following.

```
    - enum34  # [py<34]
```